### ISSUE TITLE
Creating config file

### ISSUE THREAD

**Issue by PLNReynolds** [ucdavisdatalab/workshop_intro_to_remote_computing#4]
Add section before "1.5.1. The config File" re: how to create a config file if they don't have one (and for diff os)?

**Comment by nick-ulle** [comment]
There's no need for this to be a separate step -- running `nano filename` or `vi filename` automatically creates the file. However, we do need to make sure we're emphasizing this to students (and possibly add a callout about it to the reader).

**Comment by cstahmer** [comment]
Agreed.  And we should talk about whether Wesley’s introduction of Touch is really relevant.  Touch does have its use.  But I think he complicated matters by introducing it in his session just running vi or nano would create the files he was creating automatically if they don’t already exist.


On Nov 15, 2024, at 11:44 AM, Nick Ulle ***@***.***> wrote:



There's no need for this to be a separate step -- running nano filename or vi filename automatically creates the file. However, we do need to make sure we're emphasizing this to students (and possibly add a callout about it to the reader).
